make_functions.sh

```diff
@@ -47,7 +48,29 @@ build_image() {
   cmd_build_args=$(_build_args_builder "$build_args")
   cmd_tags=$(_tag_argument_builder "$reg_paths" "$tags")
 
+  if [[ "${is_local}" -eq 1 ]]; then
+    cmd_platforms=""
+  else
+    cmd_platforms="--platform \"${platforms}\""
+  fi
+
   echo "Building ${image} for build args \"$build_args\" and platforms \"$platforms\""
-  docker buildx build --platform "$platforms" ${cmd_build_args} \
+  echo ${cmd_platforms}
+  docker buildx build ${cmd_platforms} ${cmd_build_args} \
     -f "$dockerfile" . ${cmd_tags} -o "$output" --progress=auto
 }
+
+import_microk8s() {
```

Comment:
can we call this something like microk8s_image_update() or something

Reply:
sure, next PR